rcsb/utils/dictionary/DictMethodEntityInstanceHelper.py

```diff
@@ -758,6 +760,8 @@ def buildEntityInstanceFeatures(self, dataContainer, catName, **kwargs):
             cisPeptideD = self.__ssU.getCisPeptides(dataContainer)
             for cId, cL in cisPeptideD.items():
                 for (asymId, begSeqId, endSeqId, modelId, omegaAngle) in cL:
+                    if str(modelId) != repModelId:
+                        continue
```

Comment:
Ideally, it would also be good to turn off calculations for non-representative models at source (e.g., ssU.getCisPeptides, commonU.getUnobservedPolymerResidueInfo, commonU.getUnobservedPolymerAtomInfo, __commonU.getCompModelLocalScores).

Reply:
Thanks @brindakv. Do you think this should be implemented now, or done as a separate ticket later on? Is there a benefit to keeping the calculations around for a little longer after this loading change is put forward?

Reply:
My suggestion would be to add similar "if" statements in the appropriate places to turn off calculations for non-representative models. Retaining the same here would be a fall-back that only suppresses the loading. 

It is more efficient to turn off the calculations rather than only suppressing the loading, especially for instance features. I don't think the changes would be extensive. But if you would like to handle these in separate PRs, that is ok. Perhaps create tickets in the corresponding repos to keep track of it.